bigframes/display/anywidget.py

```diff
@@ -237,6 +259,20 @@ def _set_table_html(self) -> None:
             )
             return
 
+        # Apply sorting if a column is selected
+        df_to_display = self._dataframe
+        if self.sort_column:
+            df_to_display = df_to_display.sort_values(
```

Comment:
Could we file a bug and a TODO to support sorting by the index column(s) as well? For those, we'd use sort_index and I thing the "levels" argument.